packages/cozy-procedures/src/redux/personalDataSlice.js

```diff
@@ -1,18 +1,48 @@
 import { createSlice } from 'redux-starter-kit'
 import get from 'lodash/get'
 
+import { Contact } from 'cozy-doctypes'
+
 const personalDataSlice = createSlice({
-  initialState: null,
+  initialState: {
+    data: {},
+    loading: false,
+    error: ''
+  },
   slice: 'personalData',
   reducers: {
     init: (state, action) => {
-      return Object.keys(action.payload).reduce((acc, fieldId) => {
-        acc[fieldId] = undefined
-        return acc
-      }, {})
+      return {
+        data: Object.keys(action.payload).reduce((acc, fieldId) => {
```

Comment:
We could just change the value of `state.data` here since there is now an initial state :)

Reply:
hum, imagine the component is unmounted but `error` is not empty. It's probably safer to ensure that if we mount the component again, `error` is empty and `loading` is false, right?

Reply:
Yeah actually that's a good point 👍